### PR TITLE
edit cmd/kube-apiserver/app/server.go to structured Log

### PR DESCRIPTION
**Pull request by LinshanYu**
/kind documentation

edit cmd/kube-apiserver/app/server.go to structured Log，https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md

### CODE REVIEW

File: cmd/kube-apiserver/app/server.go [hunk 1]
```diff
@@ -178,7 +178,7 @@ cluster's shared state through which all other components interact.`,
 // Run runs the specified APIServer.  This should never exit.
 func Run(completeOptions completedServerRunOptions, stopCh <-chan struct{}) error {
 	// To help debugging, immediately log version
-	klog.Infof("Version: %+v", version.Get())
+	klog.InfoS("Version: %+v", version.Get())
```
Comment (yangjunmyfm192085) [inline cmd/kube-apiserver/app/server.go:181]: Is it correct?

File: cmd/kube-apiserver/app/server.go [hunk 2]
```diff
@@ -626,7 +626,7 @@ func Complete(s *options.ServerRunOptions) (completedServerRunOptions, error) {
 				return options, fmt.Errorf("error finding host name: %v", err)
 			}
 		}
-		klog.Infof("external host was not specified, using %v", s.GenericServerRunOptions.ExternalHost)
+		klog.InfoS("external host was not specified, using %v", s.GenericServerRunOptions.ExternalHost)
```
Comment (yangjunmyfm192085) [inline cmd/kube-apiserver/app/server.go:629]: ditto

File: cmd/kube-apiserver/app/server.go [hunk 3]
```diff
@@ -661,10 +661,10 @@ func Complete(s *options.ServerRunOptions) (completedServerRunOptions, error) {
 			}
 			if s.Authentication.ServiceAccounts.ExtendExpiration {
 				if s.Authentication.ServiceAccounts.MaxExpiration < serviceaccount.WarnOnlyBoundTokenExpirationSeconds*time.Second {
-					klog.Warningf("service-account-extend-token-expiration is true, in order to correctly trigger safe transition logic, service-account-max-token-expiration must be set longer than %d seconds (currently %s)", serviceaccount.WarnOnlyBoundTokenExpirationSeconds, s.Authentication.ServiceAccounts.MaxExpiration)
+					klog.InfoS("service-account-extend-token-expiration is true, in order to correctly trigger safe transition logic, service-account-max-token-expiration must be set longer than %d seconds (currently %s)", serviceaccount.WarnOnlyBoundTokenExpirationSeconds, s.Authentication.ServiceAccounts.MaxExpiration)
```
Comment (yangjunmyfm192085) [inline cmd/kube-apiserver/app/server.go:664]: ditto

File: cmd/kube-apiserver/app/server.go [hunk 4]
```diff
@@ -661,10 +661,10 @@ func Complete(s *options.ServerRunOptions) (completedServerRunOptions, error) {
...
 				}
 				if s.Authentication.ServiceAccounts.MaxExpiration < serviceaccount.ExpirationExtensionSeconds*time.Second {
-					klog.Warningf("service-account-extend-token-expiration is true, enabling tokens valid up to %d seconds, which is longer than service-account-max-token-expiration set to %s seconds", serviceaccount.ExpirationExtensionSeconds, s.Authentication.ServiceAccounts.MaxExpiration)
+					klog.InfoS("service-account-extend-token-expiration is true, enabling tokens valid up to %d seconds, which is longer than service-account-max-token-expiration set to %s seconds", serviceaccount.ExpirationExtensionSeconds, s.Authentication.ServiceAccounts.MaxExpiration)
 				}
```
Comment (yangjunmyfm192085) [inline cmd/kube-apiserver/app/server.go:668]: ditto

File: cmd/kube-apiserver/app/server.go [hunk 5]
```diff
@@ -661,10 +661,10 @@ func Complete(s *options.ServerRunOptions) (completedServerRunOptions, error) {
...
+					klog.InfoS("service-account-extend-token-expiration is true, in order to correctly trigger safe transition logic, service-account-max-token-expiration must be set longer than %d seconds (currently %s)", serviceaccount.WarnOnlyBoundTokenExpirationSeconds, s.Authentication.ServiceAccounts.MaxExpiration)
 				}
 				if s.Authentication.ServiceAccounts.MaxExpiration < serviceaccount.ExpirationExtensionSeconds*time.Second {
-					klog.Warningf("service-account-extend-token-expiration is true, enabling tokens valid up to %d seconds, which is longer than service-account-max-token-expiration set to %s seconds", serviceaccount.ExpirationExtensionSeconds, s.Authentication.ServiceAccounts.MaxExpiration)
+					klog.InfoS("service-account-extend-token-expiration is true, enabling tokens valid up to %d seconds, which is longer than service-account-max-token-expiration set to %s seconds", serviceaccount.ExpirationExtensionSeconds, s.Authentication.ServiceAccounts.MaxExpiration)
```
Comment (yangjunmyfm192085) [inline cmd/kube-apiserver/app/server.go:667]: ditto

File: cmd/kube-apiserver/app/server.go (hunk 2 above)
Comment (lala123912) [reply cmd/kube-apiserver/app/server.go:629]: A capital letter  is suggested, and the format is error. Pls start after referring to  https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1602-structured-logging and https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md

**Comment by roycaihw** [comment]
Is there an umbrella issue summarizing what packages we plan to migrate?

Instead of opening one PR for each file, could you combine them into slightly more coarse grained changes (25-100 loc), and have more meaningful commit messages (e.g. "migrate xxx package to structured logging", or "migrate xxx directory to structured logging")? That way it's easier for us to comment on repeating bugs (e.g. the format error that @yangjunmyfm192085 and @lala123912 pointed out), and have better control of the commit history. 

/hold
Please let me know when you're ready

**Comment by yangjunmyfm192085** [comment]
/wg structured-logging
@LinshanYu, Could you take a look?

**Pull request closed without merging**